90zfsbootmenu/module-setup.sh

```diff
@@ -162,6 +162,25 @@ install() {
   inst_hook initqueue/settled 99 "${moddir}/hook/zfsbootmenu-ready-set.sh" || _ret=$?
   inst_hook initqueue/finished 99 "${moddir}/hook/zfsbootmenu-ready-chk.sh" || _ret=$?
 
+  # If tracing is enabled, build in the full-weight profiling library
+  if [ -n "${zfsbootmenu_enable_tracing}" ]; then
+    inst_simple "${moddir}/profiling/profiling-lib.sh" "/lib/profiling-lib.sh"
+
+    # shellcheck disable=SC2154
+    cat << EOF >> "${initdir}/etc/profiling.conf"
+export trace_term=${trace_term}
+export trace_baud=${trace_baud}
```

Comment:
This means that `trace_term` and `trace_baud` need to be configured at module setup time. Is that intended? If so, I recommend `zfsbootmenu_trace_term` and `zfsbootmenu_trace_baud`.

Reply:
Done - I've normalized the control varriables here to `zfsbootmenu_trace_` .